### ISSUE TITLE
Running JAVA Class Files

### ISSUE THREAD

**Issue by SciFunGYWS** [edpegg/CutTheKnot-Java#1]
Hi,

I'm a naive user of JAVA language. Can you please guide me on how should I run that in ECLIPSE tool. I have downloaded the classes.zip and want to run the applets.

Thank you,
Jay,
Center for Creative Learning,
Indian Institute of Technology, Gandhinagar.

**Comment by Robin-Andrews** [comment]
Agreed it would be really helpful to have some instructions for non Java programmers on how to get these apps running. I've tried unzipping and found the folder I want  - e.g. `bicolor` and running `java bicolor` from that folder, but that doesn't work (I do have Java installed and in my path). I don't have the big picture knowledge to know why or if it's even possible to get these apps running locally. Some instructions would be appreciated.

**Comment by HACP** [comment]
Hey @Robin-Andrews and @SciFunGYWS I tried a few ways to run the java classes and failed. I found however that this chrome extension allows you to run the applets in chrome as well as other legacy java apps: https://chrome.google.com/webstore/detail/cheerpj-applet-runner/bbmolahhldcbngedljfadjlognfaaein/related?hl=en hope it helps.

**Comment by mayd** [comment]
Why don't you release the Java source code?  Otherwise this is pretty useless.

**Comment by joseluis** [comment]
Hi. I ran into this same problem while trying to use the archived Cut-the-Knot applets, so I ended up making a small helper tool for it:

https://github.com/joseluis/ctk-applet

It prepares the published `classes` archive once, then lets you run applets again from their original Cut-the-Knot page URLs with `appletviewer`.

I’m leaving it here in case it saves someone else the same trouble.
